### ISSUE TITLE
Allow photos to be uploaded for each talk

### ISSUE THREAD

**Issue by toolness** [OpenNews/srccon-schedule#2]
A lot of the talks used big post-its and whiteboards for notes, and people were taking photos of them at the end of the talks. I thought it might be neat if people (or perhaps a designated archivist person) were able to upload such photos to the schedule so that they could easily be found after the talk on the session's details page.

Just a thought, maybe it's already been addressed in some way. Feel free to close!

**Comment by ryanpitts** [comment]
Hmm, this mayyyybe gets beyond scope for the schedule app itself, but it's interesting to think about.

**Comment by toolness** [comment]
Yeah, totally reasonable :)  I suppose another option might be to have an "official" hashtag advertised for the session so that if people want, they can maybe tweet pics of the session w/ that hashtag, and that could just be displayed on the session's details page? Hmm.
